hil/Cargo.toml

```diff
@@ -12,15 +12,18 @@ rust-version.workspace = true
 
 [dependencies]
 camino = "1.1.6"
-clap = { version = "4", features = ["derive"] }
+clap = { workspace = true, features = ["derive"] }
 cmd_lib = "1.9.3"
-color-eyre = "0.6"
+color-eyre.workspace = true
+ftdi-embedded-hal = { version = "0.22.0", features = ["libftd2xx-static"] }
```

Comment:
should we set this dependency at the workspace level because the cone is also using it. 
otherwise i'm gonna do it in my PR